### PR TITLE
Add more logging to the update pipeline.

### LINKED ISSUE #220: Update pipeline: Add more logging

### PR DESCRIPTION
**Pull request by AndriySvyryd**
Fixes #220

### CODE REVIEW

File: src/EFCore.Relational.Specification.Tests/TestUtilities/TestSqlLoggerFactory.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@ public class TestSqlLoggerFactory : ILoggerFactory
         private const string FileNewLine = @"
 ";
 
+        // ReSharper disable once InconsistentNaming
         private static readonly string EOL = Environment.NewLine;
```
Comment (smitpatel) [inline src/EFCore.Relational.Specification.Tests/TestUtilities/TestSqlLoggerFactory.cs]: Change name to _eol

File: tools/Resources.tt [hunk 2]
```diff
@@ -148,7 +148,7 @@ namespace <#= model.Namespace #>
         var rootNamespace = (string)project.Properties.Item("RootNamespace").Value;
         var resourceDir = Path.GetDirectoryName(resourceFile);
         var projectDir = (string)project.Properties.Item("FullPath").Value;
-        var resourceNamespace = rootNamespace + "." + resourceDir.Substring(projectDir.Length)
+        var resourceNamespace = rootNamespace + resourceDir.Substring(projectDir.Length)
```
Comment (smitpatel) [inline tools/Resources.tt]: Interesting. I had to add "." to correctly run the tool. Is it working correctly for you? Could be a random breaking change in VS.

Comment (AndriySvyryd) [reply tools/Resources.tt]: Yeah, I get an extra "." on VS 15.4.5

Comment (smitpatel) [reply tools/Resources.tt]: Breaking change in 15.5
I miss "." while running template in 15.5.1 & 15.6 preview 2
Upto you what you want to do. :trollface:

Comment (AndriySvyryd) [reply tools/Resources.tt]: Resharper is still slow on 15.5, so I think we should wait a bit before making the whole team to upgrade.

**Comment by smitpatel** [comment]
LGTM. :shipit:

**Pull request merged**